[2022-11-04T19:02:20.141+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-11-04T19:02:20.177+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-11-04T19:02:20.178+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:02:20.180+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-04T19:02:20.183+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:02:20.233+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): TASK1> on 2022-05-13 00:00:00+00:00
[2022-11-04T19:02:20.246+0000] {standard_task_runner.py:55} INFO - Started process 4722 to run task
[2022-11-04T19:02:20.256+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'orquestacion', 'TASK1', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/5.1-orquestacion.py', '--cfg-path', '/tmp/tmp0n_fksfs']
[2022-11-04T19:02:20.257+0000] {standard_task_runner.py:83} INFO - Job 40: Subtask TASK1
[2022-11-04T19:02:20.391+0000] {task_command.py:376} INFO - Running <TaskInstance: orquestacion.TASK1 scheduled__2022-05-13T00:00:00+00:00 [running]> on host c7d12af55512
[2022-11-04T19:02:20.534+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=orquestacion
AIRFLOW_CTX_TASK_ID=TASK1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-13T00:00:00+00:00
[2022-11-04T19:02:20.550+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-04T19:02:20.551+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "SLEEP 2 && ECHO 'TASK1'"]
[2022-11-04T19:02:20.574+0000] {subprocess.py:86} INFO - Output:
[2022-11-04T19:02:20.600+0000] {subprocess.py:93} INFO - /bin/bash: line 1: SLEEP: command not found
[2022-11-04T19:02:20.603+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-11-04T19:02:20.624+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-11-04T19:02:20.635+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=orquestacion, task_id=TASK1, execution_date=20220513T000000, start_date=20221104T190220, end_date=20221104T190220
[2022-11-04T19:02:20.668+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 40 for task TASK1 (Bash command failed. The command returned a non-zero exit code 127.; 4722)
[2022-11-04T19:02:20.706+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-11-04T19:02:20.772+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
